client-apis/encrypt.md

```diff
@@ -67,24 +72,31 @@ A bound on the length of the [plaintext](#plaintext) to encrypt.
 This behavior MUST output the following if the behavior is successful:
 
 - [message](#message.md)
-
-This behavior MAY output the following:
```

Comment:
This was originally included as MAY in order to "support" MKPs/MKs. However, in practice we've found the language confusing. Because this specification is scoped to describing encryption strictly using keyrings, I fixed this up to make it clear that we should always be outputting the keyring trace.

Not necessarily in scope for this PR, so if there is any dissent here I'm happy to practice good PR hygiene and get this done in a separate PR.